src/stack_aggregator.py

```diff
@@ -26,6 +26,9 @@ def get_recommended_version(ecosystem, name, version):
         .format(eco=ecosystem, pkg=name)
     payload = {'gremlin': query}
     result = execute_gremlin_dsl(url=GREMLIN_SERVER_URL_REST, payload=payload)
+    logger.info("Ecosystem {eco}, name {name}, version {version}".format(
```

Comment:
this will add too many logs. I think this can be removed.

Reply:
fixed